### PR TITLE
Add CI and badge

### PR DESCRIPTION
**Pull request by ijnek**
This PR enables testing against ROS 2 Rolling in this repo, and changes made to get tests passing are **well-separated by commit**.

~~There are still some tests that are failing, (most of them are related to changing C-style casting to C++ style casting), but those ones I will put in a follow-up PR.~~

Once #2 and #3 arem merged first, all tests should pass.

It would be good if you could check if the changes in commit https://github.com/ros-sports/hlvs_player/pull/4/commits/b76884d6d002269e942b946861d61c852648d63d didn't break anything.

### CODE REVIEW

**Comment by SammyRamone** [comment]
Looks good to me, but I did not test it. Maybe @jgueldenstein can test it with the new Webots version?

**Comment by ijnek** [comment]
There were a few missing dependencies to be fixed up over the last few commits. Now, we should see green CI, once merged!

File: LICENSE
Comment (jaagut) [inline LICENSE:189]: Is this intentional? Other files still use `2022 RoboCup Federation, Inc.` as copyright

Comment (ijnek) [reply LICENSE:189]: ``Copyright [yyyy] [name of copyright owner]`` is the default for the Apache License V2.0, and I don't think it is intended to be changed in the LICENSE file.

Which other files do I have to change? The source code files should start with ``Copyright 2022 RoboCup Federation, Inc.`` or an appropriate author.

**Comment by ijnek** [comment]
@jaagut Could I ask for another review please?

**Review by Flova — APPROVED** [review]
I tested the code and it is broken in some parts, but this is a general issue that I will address in another PR. This PR looks good to me.

**Comment by ijnek** [comment]
I'm going ahead and merging this with Flova's review, despite him being not a maintainer for this repo.

**Pull request merged**
